actions/workflows/create_vm_role.yaml

```diff
@@ -28,16 +39,16 @@ st2cd.create_vm_role:
             input:
                 key: ami_<% $.distro %>
             on-success:
-                - check_cname_if_it_already_exists
```

Comment:
just to clarify, i feel the task name is over descriptive.  check_cname is pretty descriptive to me already.